### PR TITLE
docs: clarify guarantees of the File Chooser API

### PR DESCRIPTION
**Pull request by alyssais**
The application should not depend on where the URL points, or how access to the file is granted, just that access has been granted at that URL.

@matthiasclasen, I've set you as the commit author since I used the wording you suggested on Matrix.

### CODE REVIEW

File: data/org.freedesktop.portal.FileChooser.xml [hunk 1]
```diff
@@ -28,8 +28,7 @@
       backend will present the user with a file chooser dialog.
 
       The selected files will be made accessible to the application
-      via the document portal, and the returned URI will point
-      into the document portal fuse filesystem in ``/run/user/$UID/doc/``.
+      (which may involve adding it to the document portal).
```
Comment (GeorgesStavracas) [inline data/org.freedesktop.portal.FileChooser.xml:31]: Maybe add a link to the Document portal here?

Comment (alyssais) [reply data/org.freedesktop.portal.FileChooser.xml:31]: How would I do that?  If I write ``:ref:`Document portal<org.freedesktop.portal.Document>` `` in this section, it just generates `<span class="xref std std-ref">Document portal</span>`, with no hyperlink.

Comment (alyssais) [reply data/org.freedesktop.portal.FileChooser.xml:31]: Ah, got it — it's the Document**s** portal, in the plural.

Comment (alyssais) [reply data/org.freedesktop.portal.FileChooser.xml]: Done.

**Review by GeorgesStavracas — APPROVED** [review]

**Pull request merged**